script/keyring_validate.sh

```diff
@@ -31,6 +31,9 @@ function bail() {
 # Temporary GPG keyring for messing around with.
 tmp_gpgdir="$(mktemp -d --tmpdir "$project-validate-tmpkeyring.XXXXXX")"
 trap 'rm -r "$tmp_gpgdir"' EXIT
+function gpg_user() {
+	gpg --homedir="$tmp_gpgdir" --no-default-keyring --keyring="$username.keyring" "$@"
+}
```

Comment:
I don't really like having a loop variable be used like this -- maybe make it the first argument to `gpg_user`?

```suggestion
function gpg_user() {
	local username="$1"
	shift
	gpg --homedir="$tmp_gpgdir" --no-default-keyring --keyring="$username.keyring" "$@"
}
```

Reply:
Thanks, fixed!